src/main.cpp

```diff
@@ -95,6 +95,9 @@ map<uint256, COrphanTx> mapOrphanTransactions GUARDED_BY(cs_orphancache);
 map<uint256, set<uint256> > mapOrphanTransactionsByPrev GUARDED_BY(cs_orphancache);
 void EraseOrphansFor(NodeId peer) EXCLUSIVE_LOCKS_REQUIRED(cs_orphancache);
 
+// BU: start block download at a low number in case our peers are slow when we start  
+static unsigned int MAX_BLOCKS_IN_TRANSIT_PER_PEER = 1;       
+static unsigned int BLOCK_DOWNLOAD_WINDOW = 8;
```

Comment:
I know that its weird that all these constants are defined in main.h.  I would not have done it that way.  However, there is value in keeping all the constants together as much as possible, and in not moving things around for ease of rebase.  So please keep these in their original location in main.h :-(